__tests__/app.test.js

```diff
@@ -100,6 +101,39 @@ describe('GET /api/', () => {
 
     });
 
+    describe('GET /api/articles', () => {
+        test('200: responds with an articles array of article objects, each of which should have the following properties: author, title, article_id, topic, created at, votes, comment_count', () => {
+            return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body : { articles } }) => {
+                expect(articles).toHaveLength(12)
+                articles.forEach((article) => {
+                    expect(article).toEqual(
+                      expect.objectContaining({
+                        article_id: expect.any(Number),
+                        title: expect.any(String),
+                        topic: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
```

Comment:
In the ticket description, it wants us to exclude the body for this endpoint so will have to make sure that happens in your model